site/_posts/2021-01-07-gnap-next-gen-oauth.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: blog-post
+title: GNAP, the next generation of OAuth
+description: What is GNAP and why should you care?
```

Comment:
"What is GNAP, why should you care, and how it relates to OAuth"... or some other way to incorporate "OAuth" into the description